sql/core/src/test/scala/org/apache/spark/sql/test/SQLTestUtils.scala

```diff
@@ -306,6 +306,11 @@ private[sql] trait SQLTestUtils
     val fs = hadoopPath.getFileSystem(spark.sessionState.newHadoopConf())
     fs.makeQualified(hadoopPath).toUri
   }
+
+  def makeQualifiedPath(path: Path): URI = {
```

Comment:
actually we don't need this, we already have `def makeQualifiedPath(path: String): URI`. let's remove it in follow up